README.docker.md

```diff
@@ -0,0 +1,26 @@
+# Running in Docker
```

Comment:
Should this be in our normal README? so people can find it easily?

(or this a standard way people do it in Docker?)